File: Examples/CSharp/DWG-Drawings/GetBlockAttributeValue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Aspose.CAD;
namespace Aspose.CAD.Examples.CSharp.DWG_Drawings
{
    public class GetBlockAttributeValue
    {
        public static void Run()
        {
            //ExStart:GetBlockAttributeValue
            // The path to the documents directory.
            string MyDir = RunExamples.GetDataDir_DWGDrawings();
            string sourceFilePath = MyDir + "sample.dwg";
            // Load an existing DWG file as CadImage.
            using (Aspose.CAD.FileFormats.Cad.CadImage cadImage = (Aspose.CAD.FileFormats.Cad.CadImage)Aspose.CAD.Image.Load(sourceFilePath))
            {
                System.Console.WriteLine(cadImage.BlockEntities["*MODEL_SPACE"].XRefPathName);
            }
            //ExEnd:GetBlockAttributeValue  
        }
    }
}

File: Examples/VisualBasic/DWG-Drawings/GetBlockAttributeValue.vb
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports Aspose.CAD

Namespace DWG_Drawings
    Public Class GetBlockAttributeValue
        Public Shared Sub Run()
            'ExStart:GetBlockAttributeValue
            ' The path to the documents directory.
            Dim MyDir As String = RunExamples.GetDataDir_DWGDrawings()
            Dim sourceFilePath As String = MyDir & Convert.ToString("sample.dwg")
            ' Load an existing DWG file as CadImage.
            Using cadImage As Aspose.CAD.FileFormats.Cad.CadImage = DirectCast(Aspose.CAD.Image.Load(sourceFilePath), Aspose.CAD.FileFormats.Cad.CadImage)
                System.Console.WriteLine(cadImage.BlockEntities("*MODEL_SPACE").XRefPathName)
            End Using
            'ExEnd:GetBlockAttributeValue  
        End Sub
    End Class
End Namespace
